tools/reactive-controllers/README.md

```diff
@@ -247,20 +247,24 @@ Implements the W3C ARIA roving tabindex pattern for keyboard navigation in compo
 
 ---
 
-#### SystemContextResolutionController
+### [SystemContextResolutionController](./system-context-resolution.md)
```

Comment:
This heading doesn't follow the pattern of the other controllers above it, do we want to change it back to an unlinked h4 like the others?